tests/e2e/specs/gantt.cy.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+describe('Gantt view', () => {
+  beforeEach(() => {
+    cy.visit('/#/gantt/one')
+  })
+  describe('Gantt view', () => {
+    it('Should display the mocked workflow', () => {
+      cy
+        .get('.vue-apexcharts')
+        .should('be.visible')
```

Comment:
Would be good to test something here, but admittedly it's difficult to do this without testing the internal implementation of Apex Charts which is not the purpose of these tests.

@MetRonnie any ideas?

Reply:
We have already discussed this and came to the same conclusion. We have unit tests that make sure that apex charts is given the correct input.
I did originally did have tests looking at bar length in the charts. But this was fiddly and different results from different browsers.
I think we just have to assume that apex charts displays things correctly.